tests_e2e/orchestrator/lib/agent_test_suite_combinator.py

```diff
@@ -118,160 +119,215 @@ def _next(self) -> Optional[Dict[str, Any]]:
     def create_environment_for_existing_vm(self) -> List[Dict[str, Any]]:
         loader = AgentTestLoader(self.runbook.test_suites, self.runbook.cloud)
 
-        environment: List[Dict[str, Any]] = [
-            {
-                "c_env_name": self.runbook.vm_name,
-                "c_vm_name": self.runbook.vm_name,
-                "c_location": self.runbook.location,
-                "c_test_suites": loader.test_suites,
-            }
-        ]
+        environment: Dict[str, Any] = {
+            "c_env_name": self.runbook.vm_name,
+            "c_vm_name": self.runbook.vm_name,
+            "c_location": self.runbook.location,
+            "c_test_suites": loader.test_suites,
+        }
 
         log: logging.Logger = logging.getLogger("lisa")
-        log.info("******** Environment for existing VMs *****")
-        log.info(
-            "{  c_env_name: '%s', c_vm_name: '%s', c_location: '%s', c_test_suites: '%s' }",
-            environment[0]['c_env_name'], environment[0]['c_vm_name'], environment[0]['c_location'], [s.name for s in environment[0]['c_test_suites']])
-        log.info("***************************")
+        log.info("******** Waagent: Settings for existing VM *****")
+        log.info("")
+        log.info("Settings for %s:\n%s\n", environment['c_env_name'], self._get_env_settings(environment))
+        log.info("")
```

Comment:
A previous PR changed how we log the test environments in create_environment_list, but I forgot to update create_environment_for_existing_vm; updating it now